Misc/NEWS.d/next/Core and Builtins/2019-11-08-00-36-10.bpo-38707.SZL036.rst

```diff
@@ -0,0 +1 @@
+`multiprocessing.Process` objects now retain their correct `MainThread.native_id` attribute
```

Comment:
Just a bit of markup here:

```suggestion
:class:`multiprocessing.Process` objects now retain their correct ``MainThread.native_id`` attribute.
```

Reply:
A test for this would be relatively simple – I’ll write one and add it to the PR. Good suggestion!